src/react-apps/applications/ux-editor/src/containers/Toolbar.tsx

```diff
@@ -105,6 +105,7 @@ class ToolbarClass extends React.Component<IToolbarProps, IToolbarState> {
             title: c.name,
           },
           dataModelBindings: {},
+          func: null,
```

Comment:
What purpose does this func? I can not see any changes to our addFormComponent saga that handles this? Or?

Reply:
None. Forgot to delete it.